src/apps/service/apps/de/listings.clj

```diff
@@ -494,6 +495,13 @@
     (get-app-extra-info app-id)
     nil))
 
+(defn- format-app-documentation
+  [app-id username admin?]
+  (when admin?
+    (try+
+      (docs/get-app-docs username app-id admin?)
+      (catch [:type :clojure-commons.exception/not-found] _ nil))))
```

Comment:
(this is necessary because the get-app-docs function was written for the docs endpoint rather than for this -- if the whole app itself isn't found, the exception will be thrown elsewhere, but if the docs don't exist we just don't include them)